charts/fluentd/values.yaml

```diff
@@ -130,7 +130,8 @@ secrets:
   logzioShippingToken: ""
   logzioListener: ""
 
-secretName: logzio-logs-secret
+defaultSecretName: logzio-logs-secret
```

Comment:
Now this value is not used anywhere

Reply:
This value has been removed completely https://github.com/Shaked/logzio-helm/commit/f4864a4c70abbdfcec04f3f41b6fcc749d6024f9. I think that GH shows it only because we have a conversation open